tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/MainActivity.kt

```diff
@@ -8,46 +8,38 @@ import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.*
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Surface
-import androidx.compose.runtime.Composable
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
 import androidx.tv.foundation.lazy.list.TvLazyRow
-import band.effective.office.tv.screen.LeaderIdEvets.LeaderIdEventsScreen
-import com.example.effecticetv.ui.theme.EffecticeTVTheme
+import band.effective.office.tv.screen.leaderIdEvents.LeaderIdEventsScreen
+import band.effective.office.tv.screen.load.LoadScreen
+import band.effective.office.tv.ui.theme.EffecticeTVTheme
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.delay
 
 @AndroidEntryPoint
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
+            var isLoad by remember { mutableStateOf(false) }
+            LaunchedEffect(Unit) {
+                while(true) {
+                    delay(3000)
```

Comment:
Sliding logic: delay, state changing must be placed in `ViewModel`
All actions from UI must be provided via special call event like this: 
```kotlin
viewModel.sendEvent(LeaderIdEvent.SomeEvent)
```